File: pset6/mario/more/mario.py
# double pyramid height between 1 to 8


from cs50 import get_int


def main():

    userInput = get_positive_int()  # getting user input

    for i in range(1, userInput + 1):  # printing pyramid
        print(" " * (userInput - i) + "#" * i + "  " + "#" * i)


def get_positive_int():  # prompt user for positive integer between 1 to 8
    while True:
        n = get_int("Pyramid height: ")
        if n > 0 and n < 9:
            break
        print("Enter number between 1 to 10")
    return n


main()

File: pset1/mario/less/mario.c
//CS50x pset1 - 2020 mario/less
#include <cs50.h>
#include <stdio.h>

//This program create a right-angled pyramid between number 1 and 8.
int main(void)
{
    //prompt user for input 
    int n; 
    do
    {
        n = get_int("Height: ");
    }
    while (n > 8 || n < 1);

    //This loop controls rows.
    for (int i = 0; i < n; i++)
    {
        //This loop controls spaces
        for (int j = 1; j < n - i; j++)
        {
            printf(" ");
        }
        
        //This loop controls hashes
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");  
    }

}

